pkgs/servers/sql/patroni/default.nix

```
@@ -0,0 +1,54 @@
+{ lib, pythonPackages, fetchurl }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "patroni-${version}";
```

The `name` attr will be applied automatically.

```suggestion
  pname = "patroni";
```